proof_ipa.go

```diff
@@ -83,17 +82,11 @@ type Proof struct {
 	PostValues [][]byte
 }
 
-type SuffixStateDiff struct {
-	Suffix       byte      `json:"suffix"`
-	CurrentValue *[32]byte `json:"currentValue"`
-	NewValue     *[32]byte `json:"newValue"`
-}
-
-type SuffixStateDiffs []SuffixStateDiff
-
 type StemStateDiff struct {
-	Stem        [StemSize]byte   `json:"stem"`
-	SuffixDiffs SuffixStateDiffs `json:"suffixDiffs"`
```

Comment:
OK, now makes sense. The **PR description** has a small mistake that you might want to fix. Delete the `SuffixDiffs SuffixStateDiffs json:"suffixDiffs"` line since this field is removed.